### PR TITLE
SOC-5233 : make spaces switcher more generic by making the filtering …

### PR DESCRIPTION
**Pull request by thomasdelhomenie**
…on app configurable (it was previously hardcoded for wiki, meaning that only spaces containing the wiki app will be listed)

### CODE REVIEW

File: commons-webui-resources/src/main/webapp/javascript/eXo/commons/UISpaceSwitcher.js [hunk 1]
```diff
@@ -336,7 +341,7 @@ UISpaceSwitcher.prototype.onChooseSpace = function(spaceId, uicomponentId) {
   eval(url);
 }
 
-UISpaceSwitcher.prototype.openComboBox = function(event, spaceChooserDiv) {
+UISpaceSwitcher.prototype.openComboBox = function(event, spaceChooserDiv, appId) {
```
Comment (trongtt) [inline commons-webui-resources/src/main/webapp/javascript/eXo/commons/UISpaceSwitcher.js]: Seems this is an unnecessary change.

Comment (thomasdelhomenie) [inline commons-webui-resources/src/main/webapp/javascript/eXo/commons/UISpaceSwitcher.js]: Right.
